src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -189,7 +190,7 @@ def validator(self) -> Optional[StackValidator]:
         """Validates that the stack contains a container registry and that
         requirements are met for local components."""
 
-        def _validate_local_requirements(stack: Stack) -> Tuple[bool, str]:
+        def _validate_local_requirements(stack: "Stack") -> Tuple[bool, str]:
 
```

Comment:
extra line?

Reply:
hey, I modified this one from `Stack` to `"Stack"`, because I modified the import too to just be imported when `TYPE_CHECKING`. The `Stack` is only being used for type hinting in the file `kubeflow_orchestrator.py`, so there is no need to import the class out of `TYPE_CHECKING`. Having that said, as the import is only done when `TYPE_CHECKING`, we have to put the type hint as a string so we don't have an error when executing.